themes/default/content/docs/intro/console/organizations.md

```diff
@@ -35,65 +35,82 @@ To create an organization:
+identity provider in order to join a Pulumi organization.
 
 For example, if a Pulumi organization, is backed by a GitHub organization, then only members
 of that GitHub organization may be added to the Pulumi organization. Similarly, as soon as
 someone loses access to the GitHub organization, they will no longer have access to the
 Pulumi organization.
 
-{{% notes %}}
-See [Organization Roles]({{< relref "/docs/intro/console/organization-roles" >}}) or
-[Adding New Identities]({{< relref "/docs/intro/console/accounts#adding-new-identities" >}})
-for more information.
-{{% /notes %}}
+In addition, a Pulumi organization can be backed by a [SAML 2.0 identity provider]({{<
+relref "/docs/guides/saml" >}}).
 
-The following table shows the relationship between a Pulumi organization and third-party
-groupings.
+### Changing Identity Providers
 
-| Pulumi | GitHub | GitLab | Bitbucket |
-|--------|--------|--------|--------|
-| Organization | [Organization](https://github.com/collab-uniba/socialcde4eclipse/wiki/How-to-setup-a-GitHub-organization,-project-and-team) | [Group](https://docs.gitlab.com/ce/user/group/)| [Workspace](https://bitbucket.org/blog/introducing-workspaces) |
+Every organization is backed by an identity that governs the membership to your organization.
+By default, when you create a new Pulumi organization, it uses the Pulumi identity provider.
 
-In addition, a Pulumi organization may be backed by a [SAML 2.0 identity provider]({{<
-relref "/docs/guides/saml" >}}).
+Only organization admins can change the organization identity provider.
+
+To change an organization's identity provider:
 
-### GitHub-backed
+1. Navigate to the organization's **Settings**.
+1. Navigate to **Access Management**.
+1. Select **Change requirements**.
+
+### GitHub Identity Provider
 
-To add a GitHub-backed organization to Pulumi, an admin of the GitHub organization
-must
-first grant the Pulumi OAuth app the `read:org` scope. This can be done on GitHub's
-[Applications
-page](https://github.com/settings/connections/applications/7cf9078f3c92b17a5f0f).
+[Setting up a GitHub Organization](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch)
 
-Pulumi requires the [`read:org`
-scope](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/#available-scopes)
-in order to verify memberships within the GitHub organization. The Pulumi Console
-will not have access to any of the organization's source code, issues, or other data.
+To add a GitHub organization to Pulumi, an admin of the GitHub organization
+must first grant the Pulumi OAuth app the [`read:org` scope](https://github.com/settings/connections/applications/7cf9078f3c92b17a5f0f).
+This is required to verify memberships within the GitHub organization.
+The Pulumi Console will not have access to any of the organization's source code, issues, or other data.
```

Comment:
Perhaps we should say "Pulumi will not have access..." just to be extra clear here?  I think it's all right either way.